TriblerGUI/widgets/trustpage/js/radial.js

```diff
@@ -82,10 +80,9 @@ function angularDifference(alpha, beta) {
  * @param {number} x - The x coordinate of the vector
  * @param {number} y - The y coordinate of the vector
  * @param {number} alpha_target - the target angle to go to
- * @param {number} min_distance - the minimum distance before a force is applied
  * @returns {{x: number, y: number}} x,y in (-inf, +inf)
  */
-function radialForceVector(x, y, alpha_target, min_distance) {
+function radialForceVector(x, y, alpha_target) {
```

Comment:
This should stay in here. I made a mistake, should have used this inside the method body, will fix it in a different PR.